diff --git a/assets/javascripts/redmine_agile.js b/assets/javascripts/redmine_agile.js
index 8e3dc8b..dc1d531 100755
--- a/assets/javascripts/redmine_agile.js
+++ b/assets/javascripts/redmine_agile.js
@@ -227,10 +227,10 @@
               self.successSortable(oldStatusId, newStatusId, oldSwimLaneId, swimLaneId);
               $($item).replaceWith(data);
               estimatedHours = $($item).find("span.hours");
-              if(estimatedHours.size() > 0){
+              /*if(estimatedHours.size() > 0){
                 hours = $(estimatedHours).html().replace(/(\(|\)|h)?/g, '');
                 // self.recalculateEstimateHours(oldStatusId, newStatusId, hours);
-              }
+              }*/
             },
             error: function(xhr, status, error) {
               self.errorSortable($oldColumn, xhr.responseText);
